logs/CHANGELOG.md

```diff
@@ -24,6 +28,15 @@
 
 ## Fluent-Bit
 
+### v2.1.3 / 2023-06-21
+
+* [UPGRADE] Upgrade Fluent-bit version to 2.1.3
```

Comment:
There is 2.1.5, should we go with latest?

Reply:
2.1.3 was the latest version I could find release notes for, so we chose that one:
https://fluentbit.io/announcements/